src/webapi/window.rs

```diff
@@ -86,4 +88,17 @@ impl Window {
             ).into_reference_unchecked()
         }
     }
+
+    /// Sets a callback in requestAnimationFrame to call on next frame
+    /// Callback accepts an f64 as an argument
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)
+    pub fn request_animation_frame< F: FnMut(f64) + 'static>( &self, callback: F) -> RequestAnimationFrameHandle {
```

Comment:
Correct me if I'm wrong, but shouldn't this be `FnOnce` instead of `FnMut`, right? If that's the case then please change it. To pass an `FnOnce` you need to wrap it with `webcore::serialization::Once`.

Reply:
I wasn't sure how to take care of this part. I couldn't find an example of how to wrap a FnOnce in a Once. In order to do it directly I had to make the field public. If there's another way to do this, I'd be willing to do that :).